.github/workflows/cron-jobs.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
           # We load the dqlite libs here instead of doing through make because TICS
           # will try to build parts of the project itself
           sudo add-apt-repository -y ppa:dqlite/dev
-          sudo apt install dqlite-tools libdqlite-dev -y
+          sudo apt install dqlite-tools-v2 libdqlite1.17-dev
```

Comment:
Do we need a `-y` here as well?

Reply:
If you look carefully we already have it in: `sudo add-apt-repository -y ppa:dqlite/dev`